packages/bitcore-wallet-service/src/lib/updatestats.ts

```diff
@@ -27,12 +28,13 @@ export class UpdateStats {
     }
     uri = uri + 'readPreference=secondaryPreferred';
     console.log('Connected to ', uri);
-    mongodb.MongoClient.connect(uri, (err, db) => {
+    mongodb.MongoClient.connect(uri, { useUnifiedTopology: true }, (err, client) => {
```

Comment:
https://mongodb.github.io/node-mongodb-native/3.3/reference/unified-topology/

Nothing to change, just found out the concept of calling connect will go away soon. Was trying to research that flag

Reply:
yes, thanks for checking, will change that next.